### PR TITLE
Add descriptions into appended column list

### PR DESCRIPTION
**Pull request by zhx828**
The added descriptions include the following
- GENE_SUMMARY
- VARIANT_SUMMARY
- TUMOR_TYPE_SUMMARY
- DIAGNOSTIC_SUMMARY
- PROGNOSTIC_SUMMARY
- MUTATION_EFFECT_DESCRIPTION

This fixes https://github.com/oncokb/oncokb-annotator/issues/199

### CODE REVIEW

**Review by calvinlu3 — COMMENTED** [review]
Tests seem to be failing before able to annotate.

File: test_Annotation.py [hunk 1]
```diff
@@ -33,13 +33,15 @@
 HIGHEST_PX_LEVEL_INDEX = HIGHEST_DX_LEVEL_INDEX + 5
 UNKNOWN = 'Unknown'
 NUMBER_OF_ANNOTATION_COLUMNS = 27
+NUMBER_OF_ANNOTATION_COLUMNS_WITH_DESCRIPTIONS = 32
```
[the remark this answers is not in the document]
Comment (zhx828) [reply test_Annotation.py:36]: Makes sense

File: AnnotatorCore.py [hunk 2]
```diff
@@ -210,6 +210,9 @@ def setsampleidsfileterfile(f):
 SV_TYPE_HEADER = ['SV_CLASS_NAME', 'SV_TYPE', 'CLASS']
 SV_TYPES = ['DELETION', 'TRANSLOCATION', 'DUPLICATION', 'INSERTION', 'INVERSION', 'FUSION', 'UNKNOWN']
 
+DESCRIPTION_HEADERS = ['GENE_SUMMARY', 'VARIANT_SUMMARY', 'TUMOR_TYPE_SUMMARY', 'DIAGNOSTIC_SUMMARY',
```
[the remark this answers is not in the document]
Comment (zhx828) [reply AnnotatorCore.py:213]: @calvinlu3 I was gonna say no, but looking at the request from the user, seems like I should support it.

**Review by calvinlu3 — APPROVED** [review]
LGTM

**Pull request merged**